src/03_modules.py

```diff
@@ -10,25 +10,30 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-for arg in sys.argv:
-    print(arg)
+print("The CLI Argument is: " + str(sys.argv))
```

Comment:
you had this correct, before, you want this in a loop, instructions say one per line

Reply:
got it, thanks! Instructions were a bit confusing